how-to/use-theming/client/src/provider.ts

```diff
@@ -4,6 +4,8 @@ import { init as initialisePlatform } from "./platform";
 import type { CustomUserAppArgs, InitParams, ThemingPayload } from "./shapes";
 
 window.addEventListener("DOMContentLoaded", async () => {
+	fin.me.interop = fin.Interop.connectSync(fin.me.uuid, {});
```

Comment:
Checked with Michael and although this works the suggestion is to do it after the platform has been initialized.

Reply:
Moved to after initialization